Frameworks/CoreText/CTFont.mm

```diff
@@ -838,12 +887,11 @@ CFArrayRef CTFontCopyAvailableTables(CTFontRef font, CTFontTableOptions options)
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
```

Comment:
There's only two options, one of which is 'no options', the other is deprecated and was 'exclude synthetic data' [https://developer.apple.com/reference/coretext/ctfonttableoptions?language=objc](https://developer.apple.com/reference/coretext/ctfonttableoptions?language=objc)
I think it's close enough that we should just consider it interoperable.

---
In reply to: [83499156](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83499156) [](ancestors = 83499156)